spec/lib/task_helpers/imports/customization_templates_spec.rb

```diff
@@ -0,0 +1,124 @@
+describe TaskHelpers::Imports::CustomizationTemplate do
+  let(:data_dir) { File.join(File.expand_path(__dir__), 'data', 'customization_templates') }
+  let(:ct_file) { "existing_ct_and_pit.yaml" }
+  let(:update_file) { "update_existing.yml" }
+  let(:ct_system_file) { "system_ct.yml" }
+  let(:invalid_pt_file) { "invalid_pit_pt.yml" }
+  let(:no_pit_name_file) { "no_pit_name.yml" }
+  let(:diff_pt_file) { "pit_existing_name_new_pt.yml" }
+  let(:ct_name) { "Basic root pass template" }
+  let(:ct_desc) { "This template takes use of rootpassword defined in the UI" }
+  let(:existing_pit_name) { "RHEL-6" }
+  let(:new_pit_name) { "RHEL-7" }
+  let(:new_pit_pt) { "vm" }
```

Comment:
This is kind of a minor quibble but a lot of these are only pertinent for one context, so I think they should be moved into that context. For example like the first one I noticed `update_file` and `ct_system_file` are only used in one context each (as are a few others), so they could be moved down into them.

Arguably, since they only really pertain to the `#import` method, even the top ones like `data_dir` should be moved inside the `describe "#import" do` block, but since that's all this class is supposed to do anyway, I don't think it's that big of a deal to have ones that are used all over at this high level.